tee-worker/ts-tests/identity.test.ts

```diff
@@ -52,10 +52,7 @@ const ethereumValidationData = <LitentryValidationData>{
         Evm: {
             message: `0x${Buffer.from('mock_message', 'utf8').toString('hex')}`,
             signature: {
-                Ethereum: `0x${Buffer.from(
-                    '10ee76e356d944d17bce552a4fd0d4554ccc97dc81213f470367bd3b99c441c51',
-                    'utf8'
-                ).toString('hex')}`,
+                Ethereum: '' as HexString,
```

Comment:
may I ask why it's empty now? Then CI shouldn't pass I assume?

Reply:
I think it's a `UnexpectedMessage` , so I set it to empty and update it here:https://github.com/litentry/litentry-parachain/blob/35ba8fdc692501851f040cff679f30ffa19e0baa/tee-worker/ts-tests/identity.test.ts#L152